Jenkinsfile

```diff
@@ -0,0 +1,73 @@
+def DOCKER_IMG = "registry.fedoraproject.org/fedora:28"
+def DOCKER_ARGS = "--net=host -v /srv:/srv --privileged"
+
+// this var conveniently refers to a location on the server as well as the local dir we sync to/from
+def repo = "${env.ARTIFACT_SERVER_DIR}/repo"
+
+if (env.BUILD_TYPE != 'origin' && env.BUILD_TYPE != 'rhcos') {
+    assert false
+}
+def manifest = "host-${env.BUILD_TYPE}.json"
+
+node(env.NODE) {
+    checkout scm
+
+    docker.image(DOCKER_IMG).inside(DOCKER_ARGS) {
+        stage("Provision") {
+            sh "dnf install -y git rpm-ostree rsync openssh-clients"
+            sh "cp RPM-GPG-* /etc/pki/rpm-gpg/"
```

Comment:
This looks similar to [what's happening in the Dockerfile](https://github.com/openshift/os/blob/master/Dockerfile#L3) but in this case it looks like it's doing it on the node. Am I reading this right?

Reply:
That's correct. Since we're running in a Fedora container, we're not gonna have those keys available. I just copy them all since I'd like to generalize the Jenkinsfile to build both the `rhcos` and the `origin` variants.